crates/alkahest-renderer/src/ecs/utility.rs

```diff
@@ -174,6 +234,69 @@ impl Default for Route {
     }
 }
 
+impl Route {
+    pub fn get_command(&self, scene: &Scene, entity: Entity) -> String {
+        let mut command = String::from("route");
+        if let Some(hash) = self.activity_hash.as_ref() {
+            command = format!("{} hash {}", command, hash.0);
+        }
+        if let Some(children) = scene.entity(entity).get::<Children>() {
+            for child_ent in &children.0 {
+                let Some(pos) = scene.entity(*child_ent).get::<Transform>() else {
+                    return String::from("ERR");
```

Comment:
Use a Result (and perhaps more descriptive errors)

Reply:
Preferably use an anyhow::Result so you can just use `.context("description of what went to shitville")?` on any other fallible types (eg. results/options)